src/microbe_stage/AvailableUpgrade.cs

```diff
@@ -7,7 +7,15 @@
 /// </summary>
 public class AvailableUpgrade : IRegistryType
 {
+    private LoadedSceneWithModelInfo loadedSceneData;
+
 #pragma warning disable 169,649 // Used through reflection
+    /// <summary>
+    ///   A path to a scene to override organelle's display scene. If empty will use organelle's default model.
+    /// </summary>
+    [JsonProperty]
+    private SceneWithModelInfo overrideGraphics;
```

Comment:
Does this have to be inside the pragma warning disable? If not absolutely required, then this should be outside the pragma block.